cmake/serac-config.cmake.in

```diff
@@ -139,7 +141,7 @@ if(NOT SERAC_FOUND)
 
   # Petsc
   if(SERAC_USE_PETSC)
-    # Note: Targets not currently imported
+    include("${CMAKE_MODULE_PATH}/FindPETSc.cmake")
```

Comment:
```suggestion
    include("FindPETSc.cmake")
```

Reply:
Try using `CMAKE_CURRENT_LIST_DIR` then. `CMAKE_MODULE_PATH` is a list of directories. It just happened to be a single entry this time but normally isnt.